content/docs/gke/customizing-gke.md

```diff
@@ -89,17 +89,13 @@ To use VMs with more CPUs or RAM:
 
 To grant additional users IAM permissions to access Kubeflow:
 
-  * Add users by editing [cluster-kubeflow.yaml](https://github.com/kubeflow/kubeflow/blob/{{< params "githubbranch" >}}/deployment/gke/deployment_manager_configs/cluster-kubeflow.yaml#L74).
-
-
-By default, this file will be located at `${KUBEFLOW_SRC}/kubeflow/gcp_config/cluster-kubeflow.yaml` after your first deployment. After making changes to the file, you'll need to apply them:
+  * Users can be granted access to Kubeflow by adding the “IAP-secured Web App User” role on the GCP console [here](https://console.cloud.google.com/iam-admin/iam). Make sure you are in the same project as your Kubeflow deployment.
```

Comment:
Thanks. I changed it, but now the phrasing is the same as the section heading above:
```
To grant additional users IAM permissions to access Kubeflow:

  * To grant users access to Kubeflow, add the “IAP-secured Web App User” role on the [IAM page in the GCP console](https://console.cloud.google.com/iam-admin/iam). Make sure you are in the same project as your Kubeflow deployment.
```
Looking at the section headings above, they are a mix between a direct command ('Add GPU nodes to your cluster') with an implicit subject of you and infinitive phrases. Perhaps we change them all to start with 'Add x'?

Reply:
You're right! The page needs a bit of refactoring. For now, how about just changing the bullet point instead of the section headings? So, change this:

"To grant users access to Kubeflow, add..."

To just this:

"Add..."

Reply:
Changed it